src/template.rs

```diff
@@ -0,0 +1,56 @@
+use handlebars::{handlebars_helper, Handlebars};
+use rand::seq::SliceRandom;
+
+pub fn addhelpers(x: &mut Handlebars) {
+    handlebars_helper!(wrap: |index: usize, s: String| {
+        let mut data = s.clone();
+        if s.len() > index + 1 {
+            data.insert(index, '\n')
+        } else {
+            data.push('\n');
+        }
+        data
+    });
+    handlebars_helper!(wrap_with: |index: usize, ending: String, s: String| {
+        let mut data = s.clone();
+        if s.len() > index + 1 {
```

Comment:
similar to the comment i made in `wrap` function, i think this should wrap all lines as well